src/Microsoft.Diagnostics.Monitoring.HostingStartup/HostingStartup.cs

```diff
@@ -15,8 +18,13 @@ public void Configure(IWebHostBuilder builder)
         {
             builder.ConfigureServices(services =>
             {
-                // TODO: Gate this behind an environment variable.
-                services.AddHostedService<ParameterCapturingService>();
+                // Add a no-op singleton for easy identification that this hosting startup is active.
+                services.AddSingleton<InProcMonitoringSentinelService>();
```

Comment:
Consider using something like a `public static int InvocationCount` property on `HostingStartup` so we don't have to add an unused service to the app's composition. The tests should be able to easily see that property either directly or through reflection.